keras_nlp/models/albert/albert_classifier.py

```diff
@@ -186,9 +186,10 @@ def __init__(
         self.dropout = dropout
 
         # Default compilation
+        logit_output = self.activation == keras.activations.linear
```

Comment:
I would love to understand this better: what is the result of `activations.get(None)`? I don't see a `None` converted to `linear` anywhere in these Tasks so it seems like they must be equivalent at some level.

Reply:
Read the PR description above :), literally answers this question verbatim.

Here's the lines of tf.keras, if interested
https://github.com/keras-team/keras/blob/5849a0953a644bd6af51b672b32a235510d4f43d/keras/activations.py#L683-L684